lib/WebServer/LapTriggerWebServer.cpp

```diff
@@ -150,13 +150,35 @@ void LapTriggerWebServer::webSocketEvent(uint8_t clientId, WStype_t type, uint8_
 
         for (index = 0; index < length; ++index)
         {
-            cmd += reinterpret_cast<char *>(payload)[index];
+            char temp = reinterpret_cast<char *>(payload)[index];
+            if (';' == temp)
+            {
+                index++;
+                break;
+            }
+            else
+            {
+                cmd += temp;
+            }
+        }
+
+        for (uint8_t paramIndex = index; paramIndex < length; ++paramIndex)
```

Comment:
Do you really now that each command will always have a minimum of one parameter?

Reply:
We do not really know. But if the command finishes at the last position of the event data, then for `uint8_t paramIndex = index` , `paramIndex < length`  evaluates to false, and the for-loop is skipped. Or is it preferable to check if we have arrived the end of the event before going into the for-loop?